roles/rook/tasks/rook-jenkins-update.yml

```diff
@@ -0,0 +1,38 @@
+---
+  - name: Gather facts
+    ec2_instance_facts:
+      filters:
+        "tag:Name": "{{ master_name }}"
+        instance-state-name: running
+    register: master_metadata
+
+  - name: create date
+    shell: "date +%d-%m-%y"
+    register: date
+
+  - name: Take a backup image of the Prod-jenkins instance
+    ec2_ami:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      instance_id: "{{ master_metadata.instances[0].instance_id }}"
+      no_reboot: yes
+      wait: yes
+      wait_timeout: 3000
+      name: "{{ master_name }}-{{ date.stdout }}"
+      tags:
+        Name: "{{ master_name }}-{{ date.stdout }}"
+
+  - name: Check if container is running
+    command: ssh -i "{{ rook_key }}" ubuntu@"{{ master_metadata.instances[0].public_dns_name }}" docker ps -a | grep -i jenkins | wc -l
```

Comment:
Is this SSHing to a host other than the host where this role is getting run against?

Reply:
After chatting with Adam on Hangouts, he explained it's done this way because we're not using `ceph-sepia-secrets` for the inventory.  Which makes sense because the names/IPs may change each time a new instance is spun up.